tests/src/FunctionalJavascript/CorporateContactFormTest.php

```diff
@@ -570,15 +635,40 @@ protected function checkCorporateFieldsOnPage($max_delta = 1): void {
         $this->assertEquals($value, $element->getValue());
       }
     }
+
+    $expected_values = [];
+    $expected_values[1] = 'Spanish (http://publications.europa.eu/resource/authority/language/SPA)';
+    $expected_values[22] = 'Finnish (http://publications.europa.eu/resource/authority/language/FIN)';
```

Comment:
Same here, we should assert all and not randoms. Make constant or a data provider method.

Reply:
answered ^^